rllib/agents/ddpg/tests/test_ddpg.py

```diff
@@ -15,11 +15,7 @@ def test_ddpg_compilation(self):
         config["num_workers"] = 0  # Run locally.
 
         # Test against all frameworks.
-        for fw in ["tf", "eager", "torch"]:
-            if fw != "tf":
-                continue
-            config["eager"] = True if fw == "eager" else False
-            config["use_pytorch"] = True if fw == "torch" else False
+        for _ in framework_iterator(config, "tf"):
```

Comment:
Shouldn't the second arg be a tuple?

Reply:
It can handle both. `str` or `Tuple[str]`.